community/sig-stack-guidance/README.md

```diff
@@ -0,0 +1,58 @@
+<!---
+This is an autogenerated file!
+
+Please do not edit this file directly, but instead make changes to the
+sigs.yaml file in the project root.
+
+To understand how this file is generated, see https://git.k8s.io/community/generator/README.md
+
+for Thoth we use `podman run --rm -e WHAT -e GO111MODULE=on -e GOPROXY -v $(pwd):/go/src/app:Z golang:1.12 make -C /go/src/app generate`
+
+--->
+
+# Stack Guidance Special Interest Group
+
+Work on recommending the most effective, performant and secure software stack for user applications.
```

Comment:
I'd like to see the topic of 'actively creating prescriptions and security related information' somehow covered. @mayaCostantini @fridex

Reply:
Thanks, added in https://github.com/thoth-station/core/pull/375/commits/4ed056cb80b5d0a80935ba187cdc6a02a7466a41 :+1: